tests/test_infer/test_models/model_utils/baichuan2_7B/modeling_baichuan.py

```diff
@@ -0,0 +1,801 @@
+# Copyright 2023 Baichuan Inc. All Rights Reserved.
```

Comment:
Consider other ways of testing. I know you would like to compare the outputs of original model and the adapted one, however, moving the whole (original) model here only for the reason of testing is pretty weird and unnecessary.

Reply:
Done